common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -2675,9 +2675,9 @@
           "ANALYZE TABLE(S) ... COMPUTE STATISTICS ... <ctx> must be either NOSCAN or empty."
         ]
       },
-      "CREATE_FUNC_WITH_IF_NOT_EXISTS_AND_REPLACE" : {
+      "CREATE_ROUTINE_WITH_IF_NOT_EXISTS_AND_REPLACE" : {
         "message" : [
-          "CREATE FUNCTION with both IF NOT EXISTS and REPLACE is not allowed."
+          "CREATE PROCEDURE or CREATE FUNCTION with both IF NOT EXISTS and REPLACE is not allowed."
```

Comment:
How about: "Cannot create a routine with both IF NOT EXISTS and REPLACE specified."

Reply:
That works too

Reply:
sounds good, we don't have PROCEDURE in OSS, will update in a followup PR